src/command/helpModule.ts

```diff
@@ -1,28 +1,42 @@
-import { Message } from "discord.js";
+import { Message, MessageEmbed } from "discord.js";
 import CookiecordClient from "../client";
 import Module from "../module";
 import { command } from "./decorator";
 import { getModuleCommands } from "../util/getModuleAssociates";
 
 export default class HelpModule extends Module {
-    constructor(client: CookiecordClient) {
-        super(client);
-    }
-    @command()
-    async help(msg: Message) {
-        const CODEBLOCK = "```";
-        const prefix = await this.client.getPrefix(msg);
-        msg.channel.send(`${CODEBLOCK}
-${Array.from(this.client.modules)
-    .filter(mod => getModuleCommands(mod).length !== 0)
-    .map(
-        module => `${module.constructor.name}:
-${Array.from(this.client.commandManager.cmds)
-    .filter(c => c.module == module)
-    .map(cmd => `	${prefix}${cmd.id.split("/")[1]}`)
-    .join("\n")}`
-    )
-    .join("\n")}
-${CODEBLOCK}`);
-    }
+  constructor(client: CookiecordClient) {
+    super(client);
+  }
+
+  @command()
+  async help(msg: Message) {
+    const prefix = await this.client.getPrefix(msg);
+    const modules = Array.from(this.client.modules).filter(
+      (mod) => getModuleCommands(mod).length !== 0
+    );
+
+    const _commands = Array.from(this.client.commandManager.cmds);
+
+    const initialEmbed = new MessageEmbed()
+      .setTitle("Help")
+      .setTimestamp()
+      .setColor(process.env.COOKIECORD_EMBED_COLOR || "DEFAULT");
+
+    const embed = modules.reduce((embed, module) => {
+      const commands = _commands.filter((command) => command.module === module);
+      return embed.addField(
+        module.constructor.name,
+        commands.map((command) => {
+          const name = command.id.split("/")[1];
+          const description = command.description
+            ? `${command.description}`
+            : "No Description";
+          return `${prefix}${name}: ${description}`;
```

Comment:
Hide `${description}` if there is none instead of falling back on a default description.